tests/beta/queuing/test_beta_measure.py

```diff
@@ -13,14 +13,16 @@
 # limitations under the License.
 """Unit tests for the measure module"""
```

Comment:
This test file has minor modifications only:

* replacing BetaTensor with the standard PL tensor operation

* takes into account the changes with the MeasurementProcess class